packages/cli-kit/src/public/node/logs.ts

```diff
@@ -0,0 +1,13 @@
+import {logsFolder} from '../../private/node/constants.js'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {mkdir, writeFile} from '@shopify/cli-kit/node/fs'
+
+export const getLogsDir = logsFolder()
+
+export const createLogsDir = async (path: string): Promise<void> => {
+  await mkdir(joinPath(logsFolder(), path))
+}
+
+export const writeLog = async (path: string, logData: string): Promise<void> => {
```

Comment:
Does this append or replace the file? Maybe best to name in such a way that the behaviour is more explicit.